server/cmd/mmctl/commands/job.go

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/client"
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/printer"
+
+	"github.com/spf13/cobra"
+)
+
+var JobCmd = &cobra.Command{
+	Use:   "job",
+	Short: "Management of jobs",
+}
+
+var listJobsCmd = &cobra.Command{
+	Use:     "list",
+	Short:   "List jobs",
+	Example: "  job list",
+	Args:    cobra.NoArgs,
+	RunE:    withClient(listJobsCmdF),
+}
+
+var updateJobCmd = &cobra.Command{
+	Use:     "update [job] [status]",
+	Short:   "Update the status of a job",
+	Example: "  job update pending",
+	Args:    cobra.MinimumNArgs(2),
+	RunE:    withClient(updateJobCmdF),
+}
+
+func init() {
+	listJobsCmd.Flags().Int("page", 0, "Page number to fetch for the list of import jobs")
+	listJobsCmd.Flags().Int("per-page", 5, "Number of import jobs to be fetched")
```

Comment:
Other commands use 200 as the default value. While I think that the whole `page` and `per-page` arguments are not very user-friendly, I think it's important to stay consistent until we have a better solution.
```suggestion
	listJobsCmd.Flags().Int("per-page", DefaultPageSize, "Number of import jobs to be fetched")
```

Reply:
I think 200 is too much for jobs and I don't want to force them to enter the per-page every time. When looking for jobs you'll usually want the latest one and in the terminal they will have to scroll through a lot of results